### ISSUE TITLE
Keep and handle astropy units up to final results (DL3)

### ISSUE THREAD

**Issue by vuillaut** [cta-observatory/cta-lstchain#132]

**Comment by rlopezcoto** [comment]
we need this very badly, especially now that we are already calculating sensitivities...

**Comment by vuillaut** [comment]
Yeah it's been on my todo list for too long now but it's will require many changes.
As it's becoming urgent, I'll move it higher in my list of tasks.
